q2_diversity_lib/citations.bib

```diff
@@ -7,5 +7,37 @@ @article{faith1992conservation
   pages={1--10},
   year={1992},
   publisher={Elsevier},
-  doi={https://doi.org/10.1016/0006-3207(92)91201-3}
+  doi={10.1016/0006-3207(92)91201-3}
+}
+
+@article{pielou1966measurement,
+  title={The measurement of diversity in different types of biological collections},
+  author={Pielou, Evelyn C},
+  journal={Journal of theoretical biology},
+  volume={13},
+  pages={131--144},
+  year={1966},
+  publisher={Elsevier},
+  doi={10.1016/0022-5193(66)90013-0}
+}
+
+% TODO: Selected during a preliminary assay of the literature - must be confirmed/augmented
+@article{shannon1948communication,
+  title={A mathematical theory of communication},
+  author={Shannon, Claude E},
+  journal={The Bell System Technical Journal},
+  volume={27},
+  issue={3},
+  pages={379--423, 623-656},
+  year={1948},
+  publisher={Nokia Bell Labs},
+  doi={10.1002/j.1538-7305.1948.tb01338.x}
+}
+
+% TODO: Selected during a preliminary assay of the literature - must be confirmed/augmented
```

Comment:
This one I think we can just call "basic knowledge in the field", also this reference focuses more on the concept of the OTU, so we still don't have a reference for the idea of "counting unique things", which I suspect doesn't really exist, and is unlikely to be worth the reference. So I would drop this one.

Reply:
We're all on the same page here. Dropping Sokal.